xmind/tests/test_sheet_element.py

```diff
@@ -0,0 +1,402 @@
+
+    def getLogger(self):
+        if not getattr(self, '_logger', None):
+            self._logger = logging.getLogger('sheetElement')
+        return self._logger
+
+    def setUp(self):
+        super(TestSheetElement, self).setUp()
+        self._init_method = self._init_patch_with_name('_init', 'xmind.core.sheet.WorkbookMixinElement.__init__')
+        self._add_id_attribute = self._init_patch_with_name('_add_id_attribute', 'xmind.core.sheet.SheetElement.addIdAttribute')
+
+    def test_excessive_parameters(self):
+        _get_root_topic = self._init_patch_with_name('_get_root_topic', 'xmind.core.sheet.SheetElement._get_root_topic')
+
+        _element = SheetElement()
+
+        _parameters = [
+            ('createRelationship', 3),
+            ('_getRelationships', 0),
+            ('addRelationship', 1),
+            ('removeRelationship', 1),
+            ('getRootTopic', 0),
+            ('_get_title', 0),
+            ('getTitle', 0),
+            ('setTitle', 1),
+            ('getParent', 0),
+            ('updateModifiedTime', 0)
+        ]
+        self._init_method.assert_called_once_with(None, None)
+        self._add_id_attribute.assert_called_once_with(ATTR_ID)
+        _get_root_topic.assert_called_once()
+
+        for pair in _parameters:
+            with self.subTest(pair=pair):
+                self._test_method_by_excessive_parameters(pair, _element)
+
+    def test_init(self):
+        _get_root_topic = self._init_patch_with_name('_get_root_topic', 'xmind.core.sheet.SheetElement._get_root_topic')
+
+        SheetElement()
+        self._add_id_attribute.assert_called_once_with(ATTR_ID)
+        _get_root_topic.assert_called_once()
+
+    def test_get_root_topic_without_topic(self):
+        _get_child_nodes_by_tag_name = self._init_patch_with_name('_get_child_nodes_by_tag_name', 'xmind.core.sheet.SheetElement.getChildNodesByTagName', return_value=[])
+        _owner_workbook = Mock()
+        _get_owner_workbook = self._init_patch_with_name('_get_owner_workbook', 'xmind.core.sheet.SheetElement.getOwnerWorkbook', return_value=_owner_workbook)
+        _obj_topic_element = Mock()
+        _topic_element = self._init_patch_with_name('_topic_element', 'xmind.core.sheet.TopicElement', return_value=_obj_topic_element)
+        _append_child = self._init_patch_with_name('_append_child', 'xmind.core.sheet.SheetElement.appendChild')
+
+        _obj = SheetElement()
+        _obj._get_root_topic()
+
+        self.assertEqual(_get_child_nodes_by_tag_name.call_count, 2)
+        _get_child_nodes_by_tag_name.assert_called_with(TAG_TOPIC)
+
+        self.assertEqual(_get_owner_workbook.call_count, 2)
+
+        self.assertEqual(_topic_element.call_count, 2)
+        _topic_element.assert_called_with(ownerWorkbook=_owner_workbook)
+
+        self.assertEqual(_append_child.call_count, 2)
+        _append_child.assert_called_with(_obj_topic_element)
+
+    def test_get_root_topic_with_topic(self):
```

Comment:
I don't like the fact that you have wrong calls number. 
Please recheck your test.